apps/o2-blocks/src/todo/editor.scss

```diff
@@ -135,7 +135,7 @@ $color-accent-green: #31843f;
 }
 
 // Same for the "add item" input.
-.editor-block-list__block .wp-block-a8c-todo .add-new-item {
+.block-editor-block-list__block .wp-block-a8c-todo .add-new-item {
```

Comment:
@mmtr anyone to nudge about this? 😀

Reply:
I don't know who owns o2 blocks, but they are used in P2s so we can assume that latest Gutenberg will be only available. This will also require to generate a new build and upload it to WP.com but I think @ockham is familiar with the process.

Reply:
Same as above (let's keep the old `.editor-` selectors and move these changes to a separate PR).